lib/helpers.js

```diff
@@ -210,11 +210,15 @@ async function getApkanalyzerForOs (sysHelpers) {
  * in comparison to `adb dumpsys window` output parsing.
  * But the trust command does not work for `Swipe` unlock pattern.
  *
+ * In some Android devices (Probably around Android 10+) show `mCurrentFocus` as `NotificationShade` or `StatusBar`
+ * as the lock screen while they do not have mShowingLockscreen nor mDreamingLockscreen.
+ * Although, these `NotificationShade` and `StatusBar` could be also regular notification view.
+ *
  * @param {string} dumpsys - The output of dumpsys window command.
  * @return {boolean} True if lock screen is showing.
  */
 function isShowingLockscreen (dumpsys) {
-  return /(mShowingLockscreen=true|mDreamingLockscreen=true)/gi.test(dumpsys);
+  return /(mShowingLockscreen=true|mDreamingLockscreen=true|mCurrentFocus=.+NotificationShade|mCurrentFocus=.+StatusBar)/gi.test(dumpsys);
```

Comment:
should we be more restrictive about `.+NotificationShade` and `.+StatusBar` regexps to avoid false positives?

Reply:
for example `.+\bNotificationShade\b`

Reply:
ah, could be to tune the regex. Let me try.

Reply:
also would it be possible to split this long line?